### DISCUSSION TITLE
HTML scripts aren't synced to git?

### DISCUSSION THREAD

**Discussion by wingerse** [requarks/wiki#6456] [Help / Questions] [answered]
I'm creating raw html pages with html content and scripts. But it looks like the scripts aren't synced to git? 
I don't see any option to allow syncing scripts..

**Comment by NGPixel** [comment, marked as answer]
Planned for a future release.

**Comment by alivres** [comment]
I noticed that with 2-way sync enabled, editing a page in GitHub will remove all script content saved to the page in wiki.js after sync completes. Any chance we could fix this behavior so scripts don't get blown out when editing in GitHub?
